app/helpers/candidates_helper.rb

```diff
@@ -123,10 +123,10 @@ def placement_changed_text(event)
         "requalified the candidate on #{position_link}"
       when "reserved"
         "reserved the candidate on #{position_link}"
-      else
+      when "disqualified"
         <<~TEXT
           disqualified the candidate on #{position_link}
-          with reason <b>#{event.changed_to.humanize}</b>
+          with reason <b>#{event.properties['title'].humanize}</b>
```

Comment:
we agreed to call item "reason", no need to humanize since in table already humanized text should present
```suggestion
          with reason <b>#{event.properties['reason']}</b>
```